bazel/grpc_deps.bzl

```diff
@@ -113,6 +118,13 @@ def grpc_deps():
             url = "https://github.com/madler/zlib/archive/cacf7f1d4e3d44d871b605da3b647f07d718623f.tar.gz",
         )
 
+    if "json" not in native.existing_rules():
+        http_archive(
+            name = "json",
+            build_file = "@com_github_grpc_grpc//third_party:json.BUILD",
+            url = "https://github.com/nlohmann/json/releases/download/v3.7.1/include.zip",
```

Comment:
I've attempted to change this to use a code drop instead of a submodule, and I think I've done it in a way that will work smoothly for the import.  PLMK what you think.